etna/analysis/plotters.py

```diff
@@ -1120,7 +1145,12 @@ def plot_feature_relevance(
     ts:
         TSDataset with timeseries data
     relevance_table:
-        method to evaluate the feature relevance
+        method to evaluate the feature relevance;
+
+        * if :py:class:`~etna.analysis.feature_relevance.relevance.StatisticsRelevanceTable` table is used then relevances are normalized p-values
+
+        * if :py:class:`~etna.analysis.feature_relevance.relevance.ModelRelevanceTable` table is used then relevances are importances from some model
+
     normalized:
```

Comment:
I think it is strange that we don't use this property if `StatisticsRelevanceTable` is used. Probably we should normalize to sum up to 1 in this case too.

Reply:
Or we should mark that these value is used only for `ModelRelevanceTable`.